specs/protocol/isthmus/exec-engine.md

```diff
@@ -131,6 +132,13 @@ is applicable for scenarios where the actual withdrawals root value is not readi
 
 Withdrawals list in the block body is encoded as an empty RLP list.
 
+## Block Sealing
+
+[EIP-7251](https://eips.ethereum.org/EIPS/eip-7251) introduces a new consolidation
+[EIP-7685](https://eips.ethereum.org/EIPS/eip-7685) request with type `CONSOLIDATION_REQUEST_TYPE` (or `0x02`).
+After Isthmus, the OP Stack block sealing code must not add requests of `CONSOLIDATION_REQUEST_TYPE` if the optimism config
+is active. Note, this does not need to activate after any specific hardfork as this type does not exist pre-Pectra.
```

Comment:
a bit hard to understand imo, isn't it enough to say
requests of type `CONSOLIDATION_REQUEST_TYPE` must be ignored
?

Reply:
I think it's worthwhile to note that they can always be ignored and this rule to ignore doesn't need to be timestamp based since the type did not exist prior to pectra.

Reply:
read similar in https://github.com/ethereum-optimism/specs/pull/555, made more sense there. I like how you format the `_not_` in that one.

Reply:
Agree, it reads easier over there

Reply:
re-wrote this a bit @refcell . does it make sense?

Reply:
Think this reads well. If someone has verbiage changes they can always follow up as well to improve!